beta/kustomize/overlays/workspace/README.md

```diff
@@ -0,0 +1,20 @@
+# Workspace Overlay
+
+This section is only relevant if Google Workspace is your identity provider. If you are not connecting 1Password to Google Workspace, please proceed to [Deploy to the Kubernetes cluster](https://github.com/1Password/scim-examples/tree/master/kubernetes#deploy-to-the-kubernetes-cluster).
```

Comment:
Is this the right link? I'd expect this should link to the `/beta/kustomize/README.md`?